include/identity.php

```diff
@@ -663,6 +668,10 @@ function advanced_profile(App $a) {
 			$profile['age'] = array( t('Age:'), $age );
 		}
 
+		if ($age = age($a->profile['dob'], $a->profile['timezone'], '')) {
```

Comment:
Maybe once I did it. Now only against `develop`. Maybe I got confused because many other projects I have cloned use `master` for active development.